Classes/Outlet/OutletArgument.php

```diff
@@ -0,0 +1,243 @@
+<?php
+namespace FluidTYPO3\Flux\Outlet;
+
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+
+use TYPO3\CMS\Extbase\Mvc\Controller\MvcPropertyMappingConfiguration;
+use TYPO3\CMS\Extbase\Property\Exception\TargetNotFoundException;
+use TYPO3\CMS\Extbase\Utility\TypeHandlingUtility;
+use TYPO3\CMS\Extbase\Validation\Exception\NoSuchValidatorException;
+use TYPO3\CMS\Extbase\Validation\Validator\ConjunctionValidator;
+use TYPO3\CMS\Extbase\Validation\Validator\ValidatorInterface;
+use TYPO3\CMS\Extbase\Validation\ValidatorResolver;
+
+/**
+ * A outlet argument
+ *
+ * @api
+ */
+class OutletArgument
+{
+    /**
+     * @var \TYPO3\CMS\Extbase\Property\PropertyMapper
+     */
+    protected $propertyMapper;
+
+    /**
+     * @var MvcPropertyMappingConfiguration
```

Comment:
Am I missing something? Does the container inject the mapping conf or does the caller *set* the configuration? If it's the first, don't we miss an `@inject` annotation?